### ISSUE TITLE
[doc] 3.1.2.2.1  Unsecured Tiller Deployment

### ISSUE THREAD

**Issue by Fooshyhead** [SUSE/doc-caasp#660]
3.1.2.2.1  Unsecured Tiller Deployment

https://documentation.suse.com/suse-caasp/4.0/html/caasp-admin/_software_management.html#_unsecured_tiller_deployment

This is not pulling from the registry.  Is the path correct in the documentation?

**Comment by r0ckarong** [comment]
The attribute {helm_tiller_version} was not correctly replaced in the current version. This should have been substitutes with the correct version of the image to pull.

Please try again replacing the passage:

```
helm init \
    --tiller-tls \
    --tiller-tls-verify \
    --tiller-tls-cert tiller.crt \
    --tiller-tls-key tiller.key \
    --tls-ca-cert ca.crt \
    --tiller-image registry.suse.com/caasp/v4/helm-tiller:2.15.0 \
    --service-account tiller
```

We will update this shortly with the new release.

**Comment by r0ckarong** [comment]
Closed with: 91c8948

**Comment by Fooshyhead** [comment]
>>> Markus Napp  01/13/2020, 02:27 AM >>>The attribute {helm_tiller_version} was not correctly replaced in the current version. This should have been substitutes with the correct version of the image to pull.Please try again replacing the passage:```helm init \    --tiller-tls \    --tiller-tls-verify \    --tiller-tls-cert tiller.crt \    --tiller-tls-key tiller.key \    --tls-ca-cert ca.crt \    --tiller-image registry.suse.com/caasp/v4/helm-tiller:2.15.0 \    --service-account tiller```We will update this shortly with the new release.-- You are receiving this because you authored the thread.Reply to this email directly or view it on GitHub:https://github.com/SUSE/doc-caasp/issues/660#issuecomment-573574397
Micro Focus
